docs/principles/testing.md

```diff
@@ -0,0 +1,45 @@
+# Testing Strategy
+
+## End-to-End Testing
+
+End-to-end testing tests an entire product, service or application. Individual products should each have their own tests.
+
+### Websites tested by Cypress
+
+Websites should be tested by Cypress.
+
+- `/medicines/web`
+- `/learning/web`
+
+### APIs tested by Jest
+
+GraphQL APIs should be tested by Jest.
+
+- `/medicines/api`
+
+## Unit Testing
+
+### Helpers tested by Jest
+
+JavaScript and Typescript helpers should be tested by Jest.
```

Comment:
Should be "TypeScript" (this is *super* pedantic).

Reply:
It's amazing I did one capitalisation and not the other.